README.md

```diff
@@ -1,14 +1,18 @@
 
-Prep linux
+Build for Linux
```

Comment:
[nit] Mind please also adding osx instructions? Should just be `make clean; make` right?